src/config/config.ts

```diff
@@ -192,3 +192,18 @@ export function initializeConfiguration(){
     }
 
 }
+
+export function fetchServerConfig(){
+
+    return $.ajax({
```

Comment:
i'm using jquery ajax because superagent doesn't supply jsonp support out of box (seems crazy) . you have to install a plugin.